org_status/formatters/gitshelf.py

```diff
@@ -0,0 +1,32 @@
+import yaml
+from giturlparse import parse
+
+from org_status.formatters import RepoListFormatter
+
+
+class GitShelfFormatter(RepoListFormatter):
+    NAME = 'gitshelf'
+
+    def convert_repo_list_to_format(self, repos):
+        yml_data = {'books': []}
+
+        for repo in repos:
+            name = parse(repo.web_url).repo
+            yml_data['books'].append({'book': name,
+                                      'git': repo.web_url,
+                                      'branch': 'master'})
+
+        return yaml.dump(yml_data, default_flow_style=False)
+
+    def convert_format_to_repo_list(self, file_name):
+        repos = []
+
+        try:
+            with open(file_name, 'r') as file:
+                yml_data = yaml.load(file)
+
+                for repo in yml_data['books']:
+                    repos.append([repo['book'], repo['git']])
+                return repos
+        except FileNotFoundError:
+            print(f'unable to find file {file_name}')
```

Comment:
Don't print anything from these methods, they should not have this side effect. The best solution would be to not resolve this `Exception` here and let the caller handle this.